### ISSUE TITLE
NeoPixel issue

### ISSUE THREAD

**Issue by kaeltaz** [bigtreetech/BIGTREETECH-SKR-mini-E3#85]
![Adnotacja 2019-11-06 200118](https://user-images.githubusercontent.com/25663430/68329328-4e4c2780-00d1-11ea-9f3e-b1098e0cf7bf.jpg)

is this the correct wiring diagram? my neopixel are still light white. what could be the reason of the problem?

**Comment by loadletter** [comment]
> is this the correct wiring diagram? 

No, I was trying to reuse this output to control the hotend fan and killed U9, because it's output is connected directly to the neopixel signal output, there is no Q3 as can be seen in the [layout](https://github.com/bigtreetech/BIGTREETECH-SKR-mini-E3/blob/master/hardware/BTT%20SKR%20MINI%20E3%20V1.2/BTT%20SKR%20MINI%20E3%20V1.2top.pdf)
